firebase-firestore/src/androidTest/java/com/google/firebase/firestore/QueryTest.java

```diff
@@ -1079,25 +1081,14 @@ public void resumingAQueryShouldUseExistenceFilterToDetectDeletes() throws Excep
 
     // Resume the query and save the resulting snapshot for verification. Use some internal testing
     // hooks to "capture" the existence filter mismatches to verify them.
-    ExistenceFilterMismatchListener existenceFilterMismatchListener =
-        new ExistenceFilterMismatchListener();
-    QuerySnapshot snapshot2;
-    ExistenceFilterMismatchListener.ExistenceFilterMismatchInfo existenceFilterMismatchInfo;
-    try {
-      existenceFilterMismatchListener.startListening();
-      snapshot2 = waitFor(collection.get());
-      // TODO(b/270731363): Remove the "if" condition below once the Firestore Emulator is fixed
-      //  to send an existence filter.
-      if (isRunningAgainstEmulator()) {
```

Comment:
Is the emulator updated to accept bloom filter already? Or should we keep skipping the test while running on emulator?

Reply:
Unfortunately, we have to keep skipping the test on the emulator. The logic for that exists below on line 1119.